ep/src/lib.rs

```diff
@@ -1,13 +1,15 @@
-use psm::epid::*;
-use psm::error::*;
-use psm::consts::*;
+extern crate error;
+use error::*;
 use std::rc::Rc;
+use self::consts::*;
```

Comment:
I'm not sure if this is correct, but it made it compile. This is letting me use the stuff from the consts.rs in the same crate.